pkg/api/acknowledge.go

```diff
@@ -63,14 +63,31 @@ func (a *API) acknowledge(message slack.InteractionCallback) error {
 		user = a.pdClient.GetDefaultUser()
 	}
 
+	if len(message.OriginalMessage.Attachments) == 0 ||  message.OriginalMessage.Attachments[0].Text == "" {
+		return errors.New("slack message structure doesn't fit")
+	}
+
 	f := &clients.Filter{}
-	f.ClusterFilterFromText(message.OriginalMessage.Text)
-	f.AlertnameFilterFromText(message.OriginalMessage.Text)
+	if f.ClusterFilterFromText(message.OriginalMessage.Attachments[0].Text) != nil ||
```

Comment:
Please do sequentially and loop over attachments

Reply:
loop added